app/scripts/modules/core/src/pipeline/details/StageFailureMessage.tsx

```diff
@@ -72,9 +72,9 @@ export class StageFailureMessage extends React.Component<IStageFailureMessagePro
       const exceptionTitle = isFailed ? (messages.length ? 'Exceptions' : 'Exception') : 'Warning';
       const displayMessages =
         message || !messages.length ? (
-          <Markdown message={message || 'No reason provided.'} className="break-word" />
+          <Markdown message={message || 'No reason provided.'} />
```

Comment:
@christopherthielen @anotherchrisberry : what do you think about this. Since `break-word` is actually defined as `break-all` in our CSS error messages look a little hard to read. e.g.
![image](https://user-images.githubusercontent.com/9469228/51627857-4af7fe80-1ef8-11e9-8266-7678613a1d47.png)

Looks like there was a bit of back and forth (https://github.com/spinnaker/deck/pull/4344) on this. Happy to undo if you think this isn't the right thing to do.

ps. here is with the word break defaulted to breaking on words (after this change)
![image](https://user-images.githubusercontent.com/9469228/51627920-75e25280-1ef8-11e9-9a3a-28e65cb3e6d0.png)

Reply:
nice 👍

Reply:
It's unfortunate that we're in this situation - would be nice to use `break-word` instead of `break-all`. There's probably a better CSS solution - maybe hiding `overflow-x` is a better approach.

Let me try to find a stacktrace in an error message and see if we can do better...

Reply:
It's been long enough since we ran into whatever issues we were seeing with the `break-all` thing that I don't remember what it was. I dare you to change the CSS class for `break-word` to this:
```css
.break-word {
  overflow-wrap: break-word
}
```
as part of the PR (or a separate one) and we can see where it goes wrong...

Reply:
Challenge accepted...?
My head hurts reading the explanation of the diff on mdn :)
Let me make a separate PR